das/subscriber.go

```diff
@@ -0,0 +1,37 @@
+package das
+
+import (
+	"context"
+
+	"github.com/celestiaorg/celestia-node/header"
+)
+
+// subscriber subscribes to notifications about new headers in the network to keep
+// sampling process up-to-date with current network state.
+type subscriber struct {
+	done
+}
+
+func newSubscriber() subscriber {
+	return subscriber{newDone("subscriber")}
+}
+
+func (s *subscriber) run(ctx context.Context, sub header.Subscription, emit listenFn) {
+	defer s.indicateDone()
+	defer sub.Cancel()
+
+	for {
+		h, err := sub.NextHeader(ctx)
+		if err != nil {
+			if err == context.Canceled {
+				return
+			}
+
+			log.Errorw("failed to get next header", "Err", err)
```

Comment:
```suggestion
			log.Errorw("failed to get next header", "err", err)
```

Seems like an issue from global renaming